test/functional/basic/basic-e2e-specs.js

```diff
@@ -188,7 +188,9 @@ describe('XCUITestDriver - basics -', function () {
         const expectedTypes = [
           'syslog', 'crashlog', 'performance', 'server', 'safariConsole', 'safariNetwork'
         ];
-        (await driver.logTypes()).should.eql(expectedTypes);
+        const actualTypes = await driver.logTypes();
+        delete actualTypes.safariNetwork;
```

Comment:
does it work? expectedTypes does include `safariNetwork` item

Reply:
Let me check again. Cloud has an extraneous type.